src/pages/index.js

```diff
@@ -54,79 +65,227 @@ editProfilePopup.setEventListeners();
 const addCardPopup = new PopupWithForm(addCardModal, handleAddCardFormSubmit);
 addCardPopup.setEventListeners();
 
+const deleteConfirmPopup = new ConfirmPopup(
+  confirmDeleteModal,
+  handleConfirmAction
+);
+deleteConfirmPopup.setEventListeners();
+
+// Make a user object to store user information from the API call
+let user;
+
+// =======================
+// API Calls & Data Initialization
+// =======================
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6fccc61c-6377-4705-9a17-e90101d72c87",
+    "Content-Type": "application/json",
+  },
+});
+
+// Fetch the user info from the server
+api.getUserInfo().then((res) => {
+  const userData = {
+    _id: res._id,
+    name: res.name,
+    avatar: res.avatar,
+    about: res.about,
+  };
+
+  // Set user info in the UserInfo instance
+  user = new UserInfo(userData);
```

Comment:
* One remark hasn't been corrected https://github.com/dohstarks3/se_project_aroundtheus/pull/1#discussion_r2160404703 

`UserInfo` should accept **selectors** which should be used for **searching** elements inside the `UserInfo` class in the constructor.  

Please, check how your `UserInfo` looked like in project 8. So, the only difference is that you add the `avatar` element there